### PR TITLE
Renaming and project structure fix

### PR DESCRIPTION
**Pull request by KateMoreva**

### CODE REVIEW

File: src/main/java/org/embl/mobie/viewer/Main.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package de.embl.cba.n5;
+package org.embl.mobie.viewer;
```
Comment (tischi) [inline src/main/java/org/embl/mobie/viewer/Main.java:1]: @KateMoreva  this should be `org.embl.mobie.io` (not `viewer`).

**Review by tischi — CHANGES_REQUESTED** [review]

**Review by tischi — APPROVED** [review]
Perfect as far as I can see!
Thanks!

**Comment by KateMoreva** [comment]
> Perfect as far as I can see!
> Thanks!


@tischi, somehow I cannot rename this repo, could you please change the name to the `mobie-io`?

**Comment by tischi** [comment]
I renamed it.

**Pull request merged**